eng/pipelines/common/pack.yml

```diff
@@ -33,17 +33,63 @@ parameters:
   default: $(System.DefaultWorkingDirectory)
 
 steps:
-  - template: provision.yml
-    parameters:
-      checkoutDirectory: ${{ parameters.checkoutDirectory }}
-      poolName: ${{ parameters.poolName }}
-      provisionatorChannel: ${{ parameters.provisionatorChannel }}
-      gitHubToken: ${{ parameters.gitHubToken }}
+
+  - ${{ if ne(variables['Build.DefinitionName'], 'dotnet-maui') }}:
+    - template: provision.yml
+      parameters:
+        checkoutDirectory: ${{ parameters.checkoutDirectory }}
+        poolName: ${{ parameters.poolName }}
+        provisionatorChannel: ${{ parameters.provisionatorChannel }}
+        gitHubToken: ${{ parameters.gitHubToken }}
 
   - ${{ each step in parameters.prepareSteps }}:
     - ${{ each pair in step }}:
         ${{ pair.key }}: ${{ pair.value }}
 
+  - task: PowerShell@2
+    condition: eq(variables['PrivateBuild'], 'true')
+    displayName: Setup Private Feeds Credentials
+    inputs:
+      filePath: $(System.DefaultWorkingDirectory)/eng/common/SetupNugetSources.ps1
+      arguments: -ConfigFile $(System.DefaultWorkingDirectory)/NuGet.config -Password $Env:Token
+    env:
+      Token: $(dn-bot-dnceng-artifact-feeds-rw) 
+      
+  - pwsh: |
```

Comment:
this on provisinator yaml, but if we skip that we need to do it here to make java compiler happy building the aar